docs/source/Components/index.rst

```diff
@@ -56,6 +56,7 @@ PAT Components
 
 | The easiest way to instantiate a PAT playback object is via the example PatActor prefab.
```

Comment:
Should we maybe move this down to the more Unity centric section, and let the general overview come first?